doc/2/plugins/plugin-context/kerror/index.md

```diff
@@ -20,11 +20,12 @@ Example:
       "code": 1,
       "message": "An error occurred: %s",
       "class": "BadRequestError"
-	},
+	  },
```

Comment:
```suggestion
    },
```

Reply:
Bad indent, edit directly the file I think my suggestion is wrong too :grin:

Reply:
I blame tabulations :angry: